### PR TITLE
googlemaps cache precision fixes #2036

### LINKED ISSUE #2036: google map images have high cache miss rate
current caching of maps in this service is based on the parameters which include lat-long of the map center with a decimal precision of 12 points.

### PR DESCRIPTION
**Pull request by hamidzr**
support for limiting cache buster precision. 
sets the precision limit to 7 meaning lat or lon of `ab.1234567WX == ab.1234567YZ`

### CODE REVIEW

**Comment by brollb** [comment]
Could you add a test or two for the updates to the getGoogleParams fn?

File: test/unit/server/rpc/procedures/google-maps.spec.js [hunk 1]
```diff
@@ -58,13 +58,43 @@ describe('staticmap', function() {
...
     });
 
+    describe('getGoogleParams', function() {
+
+        const opts = {
+            center: {
+                lat: 36.2645738345627,
+                lon: -82.54322767345267,
+            },
+            width: (640 / 1),
```
Comment (brollb) [inline test/unit/server/rpc/procedures/google-maps.spec.js]: Why is this divided by 1?

File: src/server/rpc/procedures/google-maps/google-maps.js [hunk 2]
```diff
@@ -128,7 +132,10 @@ GoogleMaps.prototype._getMap = function(latitude, longitude, width, height, zoom
     // Check the cache
     this._recordUserMap(this.socket, options).then(() => {
 
-        cache.wrap(url, cacheCallback => {
+        // allow the lookups that are "close" to an already visited location hit the cache
+        const PRECISION = 7; // 6 or 5 is probabbly safe
```
Comment (brollb) [inline src/server/rpc/procedures/google-maps/google-maps.js]: not a big deal but "probably" has a typo

File: test/unit/server/rpc/procedures/google-maps.spec.js (hunk 1 above)
Comment (hamidzr) [reply test/unit/server/rpc/procedures/google-maps.spec.js]: it is showing division by scale which is one (copied from the service)

**Pull request merged**